src/screens/About/About.style.tsx

```diff
@@ -1,81 +1,31 @@
 import styled from 'styled-components';
-import { COLORS } from 'common';
-import { TeamList } from 'screens/About/About';
-import { COLOR_MAP } from 'common/colors';
-import ReactMarkdown from 'react-markdown';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+import { Heading1, Heading2, Heading3 } from 'components/Markdown';
 
-export const Wrapper = styled.div`
-  background-color: white;
-  min-height: calc(100vh - 64px);
+export const AboutHeading1 = styled(Heading1)`
```

Comment:
Will we eventually be streamlining these to lessen the need for custom headings?

Reply:
thats the plan. this page needed more spacing than our heading styles have built into them :(